kibana-reports/server/utils/validationHelper.ts

```diff
@@ -45,8 +45,14 @@ export const regexRelativeUrl = /^\/(_plugin\/kibana\/app|app)\/(dashboards|visu
 
 export const validateReport = async (
   client: ILegacyScopedClusterClient,
-  report: ReportSchemaType
+  report: ReportSchemaType,
+  basePath: String
 ) => {
+  report.query_url = report.query_url.replace(basePath, '');
+  report.report_definition.report_params.core_params.base_url = report.report_definition.report_params.core_params.base_url.replace(
+    basePath,
```

Comment:
is query_url/base_url the full url with hostname? e.g. `localhost:5601/alh/app/...` would it replace `alh` in `localhost`

Reply:
`query_url` doesn't contain hostname